sysvinit/gentoo/cloud-init-hotplug

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+
+command="/usr/lib/cloud-init/cloud-init-hotplugd"
+command_background=yes
+pidfile="/run/$RC_SVCNAME.pid"
+
+depend() {
+  before cloud-init
+}
+
+start_pre() {
+   if grep -q 'cloud-init=disabled' /proc/cmdline; then
+     ewarn "$RC_SVCNAME is disabled via /proc/cmdline."
```

Comment:
Fixed now. I was trying to use start_pre(), rather than writing a start() but it seems these is no way to signal a warning in start_pre() such that the start() function is not executed - only an error can be signally but that then produces a visible error during boot.

So I switched to using a start() function instead.